ERCS/erc-draft_fractional_nft.md

```diff
@@ -0,0 +1,348 @@
+---
+title: Fractionally Represented Non-Fungible Token Standard
+description: A standard for fractionally represented non-fungible tokens.
+author: Acme (@0xacme), Calder (@caldereth)
+discussions-to: <URL>
```

Comment:
Please open a thread in Ethereum Magician foruem

Reply:
Added in latest commit. Thanks!